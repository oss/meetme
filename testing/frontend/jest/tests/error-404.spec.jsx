import { render } from '@testing-library/react';
import Error404 from '../../../../frontend/pages/error-404.jsx';


it('should render as expected and match the snapshot', () => {
    const { container } = render(<Error404 />);
    expect(container.firstChild).toMatchSnapshot();
}); 
