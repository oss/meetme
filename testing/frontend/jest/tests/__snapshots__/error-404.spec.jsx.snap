// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render as expected and match the snapshot 1`] = `
<div
  class="flex items-center justify-center min-h-[80vh]"
>
  <div
    class="text-center"
  >
    <h1
      class="text-5xl font-bold"
    >
      404
    </h1>
    <p
      class="text-lg"
    >
      Page not found
    </p>
  </div>
</div>
`;
